planning/behavior_velocity_planner/src/scene_module/blind_spot/scene.cpp

```diff
@@ -233,8 +236,15 @@ bool BlindSpotModule::generateStopLine(
       RCLCPP_DEBUG(logger_, "No intersection enter point found.");
       return false;
     }
-    planning_utils::calcClosestIndex(
-      path_ip, intersection_enter_point_opt.get(), stop_idx_ip, 10.0);
+
+    geometry_msgs::msg::Pose intersection_enter_pose;
+    intersection_enter_pose.position = intersection_enter_point_opt.get();
+    const auto stop_idx_ip_opt =
+      motion_utils::findNearestIndex(path_ip.points, intersection_enter_pose, 10.0, M_PI_4);
+    if (stop_idx_ip_opt) {
+      stop_idx_ip = stop_idx_ip_opt.get();
+    }
+
```

Comment:
In this case, only distance threshold is given (since `intersection_enter_point_opt.get()` is not pose but point).
However, findNearestIndex with only distance threshold requires not point but pose, so `intersection_enter_pose` is created first.